include/swift/Runtime/ThreadLocal.h

```diff
@@ -20,14 +20,15 @@
 
 #include <type_traits>
 #include "ThreadLocalStorage.h"
+#include "Once.h"
 
 /// SWIFT_RUNTIME_SUPPORTS_THREAD_LOCAL - Does the current configuration
```

Comment:
Can you update the variable name in the comment?

Reply:
Pretty sure that was already fixed.